sdk/communication/communication-phone-numbers/src/generated/src/phoneNumbersClientContext.ts

```diff
@@ -32,48 +28,25 @@ export class PhoneNumbersClientContext extends coreHttp.ServiceClient {
     if (!options) {
       options = {};
     }
-
-    if (!options.userAgent) {
-      const defaultUserAgent = coreHttp.getDefaultUserAgentValue();
-      options.userAgent = `${packageName}/${packageVersion} ${defaultUserAgent}`;
-    }
-
-    // Building the request policy fatories based on the passed factories and the
-    // any required factories needed by the client.
-    if (Array.isArray(options.requestPolicyFactories)) {
-      // When an array of factories is passed in, we'll just add the required factories,
-      // in this case lroPolicy(). It is important to note that passing an array of factories
-      // to a new client, bypasses core-http default factories. Just the pipelines provided will be run.
-      options.requestPolicyFactories = [
-        lroPolicy(),
-        ...options.requestPolicyFactories
-      ];
-    } else if (options.requestPolicyFactories) {
-      // When we were passed a requestPolicyFactories as a function, we'll create a new one that adds the factories provided
-      // in the options plus the required policies. When using this path, the pipelines passed to the client will be added to the
-      // default policies added by core-http
-      const optionsPolicies = options.requestPolicyFactories([lroPolicy()]) || [
-        lroPolicy()
-      ];
-      options.requestPolicyFactories = (defaultFactories) => [
-        ...optionsPolicies,
-        ...defaultFactories
-      ];
-    } else {
-      // In case no request policy factories were provided, we'll just need to create a function that will add
-      // the lroPolicy to the default pipelines added by core-http
-      options.requestPolicyFactories = (defaultFactories) => [
-        lroPolicy(),
-        ...defaultFactories
-      ];
-    }
-
-    super(undefined, options);
-
-    this.requestContentType = "application/json; charset=utf-8";
-
-    this.baseUri = options.endpoint || "{endpoint}";
-
+    const defaults: PhoneNumbersClientOptionalParams = {
+      requestContentType: "application/json; charset=utf-8"
+    };
+
+    const packageDetails = `azsdk-js-azure-communication-phone-numbers/1.2.0-beta.2`;
```

Comment:
The `azure` after `azsdk-js-` is probably from the package-name in swagger/README.md.  Could you please use `@azure/communication-phone-numbers` instead?